plugins/pulp_python/plugins/importers/importer.py

```diff
@@ -65,11 +66,12 @@ def import_units(self, source_repo, dest_repo, import_conduit, config, units=Non
         :rtype:                list
         """
         if units is None:
-            search = criteria.UnitAssociationCriteria(type_ids=[constants.PACKAGE_TYPE_ID])
-            units = import_conduit.get_source_units(criteria=search)
+            units = chain(*repo_controller.get_unit_model_querysets(source_repo.repo_obj.repo_id,
+                                                                    models.Package))
 
+        units = list(units)
         for u in units:
-            import_conduit.associate_unit(u)
+            repo_controller.associate_single_unit(dest_repo.repo_obj, u)
```

Comment:
The platform calls it for this operation. I think the platform should probably be responsible for calling it for all operations, but that's a mess we haven't cleaned up yet.

https://github.com/pulp/pulp/blob/ed36708132e18bac4ab8b58d0b56cdb2ffe95685/server/pulp/server/managers/repo/unit_association.py#L259